config/README.md

```
@@ -0,0 +1,38 @@
+
+## Change System Name
+
+Let `X` is a SYSTEM name.
+
+- Rename `CMCONF_EXAMPLEConfig.cmake` to `CMCONF_XConfig.cmake`,
+- Rename `EXAMPLE` to `X` in `CMCONF_XConfig.cmake`,
+- Rename `EXAMPLE` to `X` in `CMLibStorage.cmake`.
+- Install `CMCONF_XConfig.cmake` as described above.
+
+
+[CMCONF_EXAMPLEConfig.cmake]: https://github.com/bacpack-system/package-tracker/example/config
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "cmconf_exampleconfig.cmake"

(MD053, link-image-reference-definitions)